backend/capellacollab/sessions/exceptions.py

```diff
@@ -72,3 +72,15 @@ def __init__(
             ),
             err_code="CONNECTION_METHOD_UNKNOWN",
         )
+
+
+class TooManyModelsToProvisionRequestedError(core_exceptions.BaseError):
```

Comment:
```suggestion
class TooManyModelsToProvisionRequestedError(core_exceptions.BaseError):
```

"Too many models to provision requested" sounds grammatically wrong, I'd prefer "Too many models requested to provision".

Reply:
Done